config/locales/en.yml

```diff
@@ -425,9 +425,10 @@ en:
           one: "%{count} site"
           other: "%{count} sites"
       index:
-        independent_organizations: "Independent Organizations"
+        independent_organizations:
+          one: " "
```

Comment:
If `""` would work just as well as `" "`, then I encourage you to remove the space.